sdk/src/Services/DynamoDBv2/Custom/DataModel/Configs.cs

```diff
@@ -305,6 +316,8 @@ public DynamoDBFlatConfig(DynamoDBOperationConfig operationConfig, DynamoDBConte
             bool consistentRead = operationConfig.ConsistentRead ?? contextConfig.ConsistentRead ?? false;
             bool skipVersionCheck = operationConfig.SkipVersionCheck ?? contextConfig.SkipVersionCheck ?? false;
             bool ignoreNullValues = operationConfig.IgnoreNullValues ?? contextConfig.IgnoreNullValues ?? false;
+            bool skipDescribeTableLoad = contextConfig.SkipDescribeTableLoad ?? false;
```

Comment:
I didn't think about this until after we did it over in https://github.com/aws/aws-sdk-net/pull/3025/files#diff-31f37a1ab8a8a2088f053530284e3276ff31440c8cb042c62a11286006d89cc4, but could we add this to `DynamoDBContextConfig` so users can set it globally?

Reply:
Added this to the global `DynamoDBContextConfig`. Updated the PR description to reflect this change and also added a unit test.